src/constants.js

```diff
@@ -1,3 +1,5 @@
 export const API_URL = "http://localhost:3000/";
 export const ADD_USER_API= `${API_URL}dev/user/registration/`
-export const GET_USERS=`${API_URL}dev/user/`
\ No newline at end of file
+export const GET_USERS=`${API_URL}dev/user/`
+export const SIGNIN_USER_API=`${API_URL}dev/user/login`
+export const GET_USER=`${API_URL}dev/user/`
```

Comment:
```suggestion
export const USER_SLUG=`${API_URL}dev/user/`
export const SIGNIN_USER_API=`${USER_SLUG}login`
```
We could change `GET_USERS` and `GET_USER` to `USER_SLUG` if there's no reason to maintain two distinct constants.  Other than that, looks good!